kfdefs/overlays/moc/smaug/opf-jupyterhub/kustomization.yaml

```diff
@@ -0,0 +1,23 @@
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: opf-jupyterhub
+
+resources:
+  - ../../../../base/jupyterhub
+
```

Comment:
we need the pvcs here

Reply:
can you create a kustomization in the `pvcs` directory and just add the `pvcs` directory here?